docs/content/concepts/resources.mdx

```diff
@@ -314,9 +324,9 @@ def my_asset(client: MyClientResource):
     return client.get_all_users()
 ```
 
-`setup_for_execution` and `teardown_after_execution` are each called once per run, per process. When using the in-process executor, this means that they will be called once per run. When using the multiprocess executor, each process' instance of the resource will be initialized and torn down.
+`setup_for_execution` and `teardown_after_execution` are each called once per run, per process. When using the in-process executor, this means that they will be called once per run. When using the multiprocess executor, each process's instance of the resource will be initialized and torn down.
 
-For more complex use-cases, you may instead override the `yield_for_execution`. By default, this context manager will call `setup_for_execution`, yield the resource, and then call `teardown_after_execution`, but you can override it to provide any custom behavior. This is useful for resources which require a context to be open for the duration of a run, such as database connections or file handles.
+For more complex use cases, you can override the `yield_for_execution`. By default, this context manager calls `setup_for_execution`, yields the resource, and then calls `teardown_after_execution`, but you can override it to provide any custom behavior. This is useful for resources that require a context to be open for the duration of a run, such as database connections or file handles.
```

Comment:
GH won't let me comment on this line - but should we move the "Defining pythonic I/O managers" section below to the IO manager page?

Reply:
Oooh, good question. I also thought this stuck out a bit and would make more sense on that other page.